### DISCUSSION TITLE
Analysis of single chain Fragment variable (scFv)

### DISCUSSION THREAD

**Discussion by d-jch** [milaboratory/mixcr#1659] [General]
Can mixcr analysis single chain Fragment variable (scFv) within which both V region of heavy and light chain are linked by a linker sequence.

**Comment by mizraelson** [comment]
Yes. If you can share a sample fastq file I will help with the command.

**Comment by d-jch** [comment]
[sample.nanopore.fq.gz](https://github.com/milaboratory/mixcr/files/15304008/sample.nanopore.fq.gz)
Here is an sample fastq from a nanopore output, filtered by expected length (1000~1240)

**Reply by mizraelson** [reply]
Can you also share the linker sequence. I will use it to split two chains. Are there UMIs or other barcodes in the data?

**Reply by d-jch** [reply]
no UMIs or other barcodes, but some vector sequences

primer1:   AGTAACGTTTGTCAGTAATTGCGGTTCTCACCC/GGGTGAGAACCGCAATTACTGACAAACGTTACT
primer2:   CCCGCTGATCTGATAACAACAGTGTAGATGTAACAAAATCGAC/GTCGATTTTGTTACATCTACACTGTTGTTATCAGATCAGCGGG
linker :   GCCGCCGCTGGAACTTGACCT/AGGTCAAGTTCCAGCGGCGGC

the IMGT/V-QUEST can do this, but not support large data~
![image](https://github.com/milaboratory/mixcr/assets/141316588/b8f57d23-3b9f-4bb6-afaf-081ff4460092)

**Reply by mizraelson** [reply]
Hi, can you please write to support@milaboratories.com
I will send you a custom solution
